packages/popups/src/Notice/toastNoticeHooks/useTopToastNoticeYSnapPoints.ts

```diff
@@ -19,10 +19,10 @@ export const useTopToastNoticeYSnapPoints = (
         /**
          * Let's be safe if initialWindowMetrics does not has the value
          */
-        return !!initialWindowMetrics?.frame.height
+        return initialWindowMetrics?.frame.height
```

Comment:
```suggestion
        return initialWindowMetrics?.frame.height != null
```

Reply:
Actually in becomes different now. 
If the height is zero it will use zero height instead of `screenDimensionsHeight` as it was before. Is it what you want?

Reply:
I would then put `> 0`, if it was the case :)